server/db.js

```diff
@@ -1,5 +1,7 @@
 const mongoose = require("mongoose");
 
+console.log(process.env.MONGO_URI)
+
```

Comment:
I would suggest you remove the debug logs

Reply:
Did you happen to catch this console log mentioned?